geode-core/src/main/java/org/apache/geode/management/internal/cli/result/AbstractResultData.java

```diff
@@ -147,11 +147,10 @@ public ResultData addAsFile(String fileName, String fileContents, String message
 
   public ResultData addAsFile(String fileName, byte[] data, int fileType, String message,
       boolean addTimeStampToName) {
-    byte[] bytes = data;
```

Comment:
I think the contents of `data` should be treated as if it were immutable in this usage, so don't make this and the following change.